tests/settings/settings.spec.test.ts

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const assert = require('assert').strict;
```

Comment:
```suggestion
import assert from 'node:assert/strict';
```

[node.js docs](https://nodejs.org/docs/latest/api/assert.html#strict-assertion-mode)